src/find/mod.rs

```diff
@@ -736,6 +736,29 @@ mod tests {
         }
     }
 
+    // Because the time when files exist locally is different
+    // from the time when Github Actions pulls them,
+    // it is difficult to write tests that limit a certain time period.
+    //
+    // For example, a Github Action may pull files from a new git commit within a few minutes,
+    // causing the file time to be refreshed to the pull time.
+    // and The files on the local branch may be several days old.
+    //
+    // So this test may not be too accurate and can only ensure that
+    // the function can be correctly identified.
+    #[test]
+    fn find_amin_cmin_mmin() {
+        let args = ["-amin", "-cmin", "-mmin"];
+        let times = ["-60", "-120", "-240", "+60", "+120", "+240"];
+
+        for (arg, time) in args.iter().zip(times.iter()) {
```

Comment:
Here you need a nested loop to get the expected result because `zip` doesn't generate the Cartesian product of `args` and `times`.